src/TCommandLine.cpp

```diff
@@ -1161,8 +1161,21 @@ void TCommandLine::spellCheckWord(QTextCursor& c)
     }
 
     QTextCharFormat f;
+    // This flag is needed to handle the selection correctly
     mSpellChecking = true;
     c.select(QTextCursor::WordUnderCursor);
+    if (bool IsWordLongEnoughForSpellCheck = (TBuffer::lengthInGraphemes(c.selectedText()) >= mpHost->mSpellCheckMinGraphemeLength); !IsWordLongEnoughForSpellCheck) {
```

Comment:
I know its not a new variable, but wouldn't `shouldSpellCheck` accomplish the same and sound nicer?

Reply:
That is something for @Kebap I think, and to be honest I am not sure that extracting the test into a `bool` and then immediately using it, is not that helpful ***when the test value is not otherwise used within the body of the `if (...) {...}`!***

OTOH The "wording" of that term IS explicit about its purpose whereas your alternative is not so clear.

:man_shrugging: